estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_view_search" model="ir.ui.view">
+        <field name="name">estate.property.view.search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Search Estate Properties">
+                <field name="name"/>
+                <field name="postcode" />
+                <field name="expected_price" />
+                <field name="bedrooms" />
+                <field name="living_area" />
+                <field name="facades" />
+                <filter string="Available" name="available" domain="['|', ('state', '=', 'new'),
+                                                                ('state', '=', 'offer_received')]"/>
```

Comment:
could have also used domain="('state', 'in', ['new', 'offer_received'])"
you can find all possible operators here https://www.odoo.com/forum/help-1/full-list-of-search-operators-46716